dp/cloud/python/magma/test_runner/tests/test_dp_with_orc8r.py

```diff
@@ -75,138 +81,103 @@ def test_cbsd_sas_single_step_flow(self):
-
-        self.given_single_step_cbsd_provisioned(builder)
-        with self.while_cbsd_is_active():
-            filters = get_filters_for_request_type('deregistration', self.serial_number)
-
-            update_request = EnodebdUpdateCbsdRequest(
-                serial_number=self.serial_number,
-                installation_param=InstallationParam(
-                    antenna_gain=DoubleValue(value=15),
-                    latitude_deg=DoubleValue(value=10.6),
-                    longitude_deg=DoubleValue(value=11.6),
-                    indoor_deployment=BoolValue(value=True),
-                    height_type=StringValue(value="agl"),
-                    height_m=DoubleValue(value=12.5),
-                ),
-                cbsd_category="a",
-            )
-
-            self.when_cbsd_is_updated_by_enodebd(update_request)
-            cbsd = self.when_cbsd_is_fetched(self.serial_number)
-            self.then_cbsd_is(cbsd, builder.with_latitude_deg(10.6).with_longitude_deg(11.6).payload)
-            self.then_message_is_eventually_sent(filters)
-
-    def test_cbsd_not_unregistered_when_coordinates_change_less_than_10_m(self):
-        builder = CbsdAPIDataBuilder()\
             .with_single_step_enabled(True) \
-            .with_serial_number(self.serial_number) \
-            .with_full_installation_param() \
             .with_cbsd_category("a")
 
-        self.given_single_step_cbsd_provisioned(builder)
-        with self.while_cbsd_is_active():
-            filters = get_filters_for_request_type('deregistration', self.serial_number)
-
-            update_request = EnodebdUpdateCbsdRequest(
-                serial_number=self.serial_number,
-                installation_param=InstallationParam(
-                    antenna_gain=DoubleValue(value=15),
-                    latitude_deg=DoubleValue(value=10.500001),
-                    longitude_deg=DoubleValue(value=11.5000001),
-                    indoor_deployment=BoolValue(value=True),
-                    height_type=StringValue(value="agl"),
-                    height_m=DoubleValue(value=12.5),
-                ),
-                cbsd_category="a",
-            )
-
-            self.when_cbsd_is_updated_by_enodebd(update_request)
-            cbsd = self.when_cbsd_is_fetched(self.serial_number)
-            self.then_cbsd_is(cbsd, builder.payload)
-            self.then_message_is_never_sent(filters)
-
-    def test_enodebd_update_cbsd(self) -> None:
-        builder = CbsdAPIDataBuilder() \
-            .with_serial_number(self.serial_number) \
-            .with_cbsd_category("b")
-        self.when_cbsd_is_created(builder.payload)
-        cbsd = self.when_cbsd_is_fetched(self.serial_number)
-        self.then_cbsd_is(
-            cbsd, builder.with_indoor_deployment(False).with_is_active(False).with_state(UNREGISTERED).payload,
-        )
-
-        req = EnodebdUpdateCbsdRequest(
+        update_request_1 = update_request_2 = EnodebdUpdateCbsdRequest(
```

Comment:
I'm not sure about this, since after this operation both `update_request_1` and `update_request2` point to the same object and later you change `update_request_2`.
Here is an example:
```
class A:
  def __init__(self):
    self.v = 0
x = y = A()
y.v = 1
print(x.v) # this prints 1 instead of 0
```

Reply:
it actually doesn't matter that they point to the same request, but I'll remove update_request_2 since in fact it's not needed